feature-flag/src/main/kotlin/com/linecorp/android/featureflag/FeatureFlagTask.kt

```diff
@@ -125,9 +125,23 @@ abstract class FeatureFlagTask : DefaultTask() {
         buildEnvironment: BuildEnvironment
     ): FeatureFlagData {
         val options = FeatureFlagOptionParser.parse(entry.option)
-        val selectors = FeatureFlagSelectorParser.parse(entry.value)
-        val evaluatedSelectors = FeatureFlagSelectorEvaluator.evaluate(selectors, buildEnvironment)
-        val value = FeatureFlagValueOptimizer.optimize(evaluatedSelectors)
+        val value = convertToFeatureFlagValue(entry.name, entry.value, buildEnvironment)
         return FeatureFlagData(entry.name, value, options)
     }
+
+    private fun convertToFeatureFlagValue(
+        flagName: String,
+        rawValue: String,
+        buildEnvironment: BuildEnvironment
+    ): FeatureFlagData.Value {
+        if (forciblyOverriddenFeatureFlags.forciblyEnabledFlags.contains(flagName)) {
```

Comment:
`Optional`: Because we pass `buildEnvironment`, we did not depend on any instance properties from these supporting functions. Don't we need to pass overridden flag sets as function arguments? It's optional because it looks overkilling.